packages/private/edtr-io/src/plugins/types/common.tsx

```diff
@@ -191,7 +191,7 @@ export function Controls(props: OwnProps) {
             title={
               maySave()
                 ? undefined
-                : 'Du musst zuerst die Lizenzbedingungen akzeptieren'
+                : 'Du musst zuerst die Lizenzbedingungen akzeptieren und die Änderungen ausfüllen'
```

Comment:
Does this text make sense? Shouldn't this also be dependent on `props.license` and `props.changes`?

Reply:
Maybe. But what I don't want is something like "You have to accept the license first" -> after accepting license: "You have to fill in the changes field"...

Reply:
Yep, I agree with you on that. But we shouldn't tell the user that he has to accept the license when the content has no license. Or even better: use the same conditions as in `maySave` (i.e. the current text if the content has both elements and both are missing. But `You have to accept the license first` if the changes have already been filled out)

Reply:
Oh right, forgot that license or changes could not exist at all.